### ISSUE TITLE
Thermal Metadata support

### ISSUE THREAD

**Issue by AndreiLazarescu** [drewnoakes/metadata-extractor-dotnet#265]
Hello,

Are you planning to add support for radiometric jpg files ?

Kind regards,
Andrei

**Comment by drewnoakes** [comment]
I've not come across them. Do you have samples and a specification?

**Comment by AndreiLazarescu** [comment]
I do not have specifications. If you are curios please check this topic: https://exiftool.org/forum/index.php?topic=11401.0

**Comment by drewnoakes** [comment]
Can you provide a sample image that you own? There's one on that thread, but using it would require contacting the author.

**Comment by AndreiLazarescu** [comment]
Yes, that's why I sent the link. I have one from a client, it is not under NDA or something but it is not my property. I do not think it is an issue using the ones on the thread for development purposes as long as you do not publish it or sell it or claim that it is yours. If it is a problem I will ask the guys on the thread, just let me know.

**Comment by drewnoakes** [comment]
Regardless of how it's achieved, we need a sample image to develop against, and probably a specification for the data you wish to extract.

**Comment by drewnoakes** [comment]
FLIR support merged today in #295. @AndreiLazarescu does that achieve what you were asking?

**Comment by AndreiLazarescu** [comment]
Hello,
Yes, that is exactly what I was looking for. Thumbs up!
Do you have a list of cameras that you support ? From the research done back then, not all vendors use the same metadata.

**Comment by drewnoakes** [comment]
Glad to hear. No list, no. I'm sure there are some cameras that aren't currently supported. If you can provide sample images (with permission to use them) it'll make it easier to support them.
